lib/suspenders/version.rb

```diff
@@ -1,8 +1,8 @@
 module Suspenders
-  RAILS_VERSION = "~> 6.1.6.1".freeze
+  RAILS_VERSION = "~> 7.0.0".freeze
   RUBY_VERSION = IO
     .read("#{File.dirname(__FILE__)}/../../.ruby-version")
     .strip
     .freeze
-  VERSION = "1.56.1".freeze
+  VERSION = "20230113".freeze
```

Comment:
@mike-burns this good?

Reply:
I think so. Honestly I haven't tried calver in a Ruby gem yet so I don't know what it'll do. But I hope this will work.

Reply:
Just noting, we opted for `yyymmdd.minor_version` to account for cases where we needed to release patches on the same day.